Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'jbuilder', '~> 2.5'
 gem 'bootsnap', '>= 1.1.0', require: false
 gem 'omniauth-github', github: 'omniauth/omniauth-github', branch: 'master'
 gem 'figaro'
+gem 'addressable'
```

Comment:
addressable gem allows me to encode the body of a post in the tone facade so I can send it to through faraday to get the tone.